app/src/main/java/com/example/trainingintive/dog_images_feature/presentation/AlbumFragment.kt

```diff
@@ -34,12 +38,40 @@ class AlbumFragment : Fragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        val binding = FragmentAlbumBinding.inflate(inflater, container, false)
+        binding = FragmentAlbumBinding.inflate(inflater, container, false)
         binding.apply {
             lifecycleOwner = viewLifecycleOwner
             recyclerView.adapter = adapter
             viewmodel = albumViewModel
         }
+        handleSwiping()
         return binding.root
     }
+
+    private fun handleSwiping() {
```

Comment:
Please try to extract it somehow to class(es), it adds too much noise to an Activity